subscriptions/sites/sample_site_cu_subscription/02_sample_site_cu_subscription.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     app: sample-site-cu-subscription
   annotations:
-    apps.open-cluster-management.io/git-path: manifests/sites/sample_site_cu
+    apps.open-cluster-management.io/git-path: manifests/sites/sample-site-cu
```

Comment:
why did you change that path?

Reply:
cause the site directory name in the ran manifest should match the cluster_name in the acm manifest. Otherwise it will replace all the sample_site_cu/du naming in kustomization.yaml

Reply:
That's what i don't understand... in our case our path is https://github.com/redhat-ztp/ztp-ran-manifests/tree/main/manifests/sites/sample_site_cu . If you create a new site then, should not be https://github.com/redhat-ztp/ztp-ran-manifests/tree/main/manifests/sites/$CLUSTER_NAME ? so it points to manifests/sites/$CLUSTER_NAME path?

Reply:
in the script the, the sed command is looking for the sample-site-cu and replace it with the new $CLUSTER_NAME . If the sed command also replace sample_site_cu it will also replace the file names in the kustomization.yaml which will required change the new cluster file names. I don't want to miss with that just keep it simple.